dietpi/dietpi-software

```diff
@@ -1406,6 +1400,7 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_CATX[$software_id]=12
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/webserver_stack/#llsp-web-stack'
 		aSOFTWARE_DEPS[$software_id]='84 87 89'
+		aSOFTWARE_CONFLICTS[$software_id]='75 76 78 79 83 85'
```

Comment:
should be `83 84`

Reply:
🤔 There are quite some things I don't like about GitHub desktop: You cannot simply rebase and then add a commit and push. It sees a conflicts between the commit you just did and every other commit that was not pushed yet. Then when you solve the conflicts, it merges base branch changes a way that they still show up as changes, even that they are present and were even coming from the base branch. So now we have unrelated changes shown in this PR... Furthermore, even that you three code comments were address, two of them do not show up as "outdated". More confusingly, they are attached to a line that looks very similar than before, if indeed the problem were not addressed, but it's the line on Lighttpd, hence it somehow changed the location of your comment in a very that can be dangerously misinterpreted...

And worst about it is that there is no way to revert this mess, once pushed. The merge commit to resolve the conflicts does not show any changes itself, even that it lead to the additional changes from the base branch now erroneously shown here. Trying to undo the commit hence gives `[!] There are no changes to commit.`.

At least I convinced GitHub desktop developers to allow amending to commits which were pushed already. But still while using GitHub desktop I regularly run into situations where it denies something that works perfectly fine with `git` CLI, where some step takes ages, respectively literally never finishes (also reported upstream, not only by me), because it seems to always try to stash/trashbin every step in a way that takes ages on larger repos, instead of allowing to simply reset to where you want, and I regularly fear that something is broken 😢.